docs/codeql/index.html

```diff
@@ -349,11 +349,12 @@ <h4 class="mb-3 text-mono text-gray-light text-normal">Company</h4>
                     </li>
                 </ul>
                 <ul class="list-style-none d-flex text-gray">
-                    <li class="mr-3">&copy; 2021 GitHub, Inc.</li>
+                    <li class="mr-3">&copy; 
+                        <script type="text/javascript">document.write(new Date().getFullYear());</script> GitHub, Inc.</li>
                     <li class="mr-3"><a
-                            href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-terms-of-service"
```

Comment:
(unrelated change: this old URL redirects to a URL without `free-pro-team` anyway)